patchwork-gui/src/main/java/net/patchworkmc/mixin/gui/MixinAbstractButtonWidget.java

```diff
@@ -30,11 +30,17 @@
 	@Shadow
 	protected int height;
 
+	protected int packedFGColor = 0;
```

Comment:
This PR needs to implement the functionality surrounding packedFGColor. See https://github.com/PatchworkMC/YarnForge/blob/af120a35c3e6951b7ef4318801b6b8dce6fc5420/patches/minecraft/net/minecraft/client/gui/widget/AbstractButtonWidget.java.patch

Reply:
Functionality has been implemented.